### PR TITLE
updated ffdl notebook using component yaml file

### PR DESCRIPTION
**Pull request by animeshsingh**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1241)

### CODE REVIEW

File: samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb [hunk 1]
```diff
@@ -88,17 +103,26 @@
...
-    "client = kfp.Client(KUBEFLOW_PIPELINE_LINK)\n",
+    "\n",
+    "EXPERIMENT_NAME = 'FfDL-Seldon Experiments'\n",
     "\n",
     "# Uncomment the below line if you want to create an experiment, \n",
     "# else we will assume the EXPERIMENT_NAME is already exist.\n",
     "\n",
-    "# exp = client.create_experiment(name=EXPERIMENT_NAME)\n",
-    "\n",
+    "#exp = client.create_experiment(name=EXPERIMENT_NAME)\n",
```
Comment (Ark-kun) [inline samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: `create_experiment` now now works the same as `get_experiment` if the experiment exists.

File: samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb [hunk 2]
```diff
@@ -4,7 +4,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# FfDL Kubeflow Pipeline Notebook demo\n"
+    "# Train and deploy with FfDL and Seldon demo\n"
+   ]
+  },
+  {
+   "attachments": {
+    "Screen%20Shot%202019-02-15%20at%204.50.17%20PM.png": {
```
Comment (Ark-kun) [inline samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: Is it possible to link to the images?

File: samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb (hunk 1 above)
Comment (animeshsingh) [reply samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: @Ark-kun fixed this then - should just be _create_experiment_

File: samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb (hunk 2 above)
Comment (animeshsingh) [reply samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: Would love to have the notebook as a unit which somebody can just take, without reference to anything out of the notebook

Comment (Ark-kun) [reply samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: @gaoning777 @hongye-sun @IronPan @vicaire WDYT?

Comment (Ark-kun) [reply samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: Hmm. GitHub does not seem to show the images:

image

Also TF seems to be using HTML IMG even for small images https://github.com/tensorflow/docs/blob/b5cb76b01b96fb936edb29ac0323a2d630d575ac/site/en/r2/tutorials/eager/tf_function.ipynb

Comment (gaoning777) [reply samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: I would vote for a link in the ipynb file and publish the image in the same directory or on the internet. 
1) I've searched throughout the web about embedding images in ipynb, people always attach a link to the image in the ipynb. 
2) Sharing a notebook vs. a directory does seem to be much different. Plus, the image can always be published in a publicly available cloud storage if the author wants to. 
3) It does not seem to be rendered correctly on the github.

Comment (animeshsingh) [reply samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb]: Ok removed the image reference for now. Will revisit later

**Pull request merged**